### ISSUE TITLE
TypeError on dowloading any asset of type "model"

### ISSUE THREAD

**Issue by Fak3** [makehumancommunity/community-plugins-assetdownload#14]
Plugin version: git master. 
Makehuman version: git master

Go to commutity asset dowloader, filter by type: "model", select any one model, click "download". The app crashes with the error:
```
showing category Community
hiding task Macro modelling
showing task Download assets
Traceback (most recent call last):
  File "/home/u1/Documents/makehuman/v1py3/plugins/8_asset_downloader/downloadtask.py", line 222, in _onFinished
    self.onFinished()
  File "/home/u1/Documents/makehuman/v1py3/plugins/8_asset_downloader/assetdb.py", line 439, in _downloadFinished
    bn = os.path.basename(fn)
  File "/usr/lib64/python3.6/posixpath.py", line 146, in basename
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

**Comment by Fak3** [comment]
plugin log:
```
[WARN ] remoteasset.py/getPertinentFileName():250 Could not find pertinent file for asset Neandertaler
[WARN ] remoteasset.py/getPertinentFileName():251 File type is model
[WARN ] remoteasset.py/getPertinentFileName():252 Files contain {'screenshot': 'http://files.makehumancommunity.org/sites/default/files/models/1168/Neandertaler.png', 'file': 'http://files.makehumancommunity.org/sites/default/files/models/1168/Neandertaler.mhm', 'thumb': 'http://files.makehumancommunity.org/sites/default/files/models/1168/Neandertaler.thumb'}
```

**Comment by Fak3** [comment]
The fix can be  as simple as change `fileForType["model"] = "mhm"` to `fileForType["model"] = "file"` at line 43: https://github.com/makehumancommunity/community-plugins-assetdownload/blob/master/8_asset_downloader/remoteasset.py#L43

But not sure if it is right thing to do on the client? Or the raname from 'file' to 'mhm' should be done on the server?

**Comment by Aranuvir** [comment]
> Or the rename from 'file' to 'mhm' should be done on the server?
I think that seems to be the case. It has to be fixed by Joel.

**Comment by joepal1976** [comment]
I've fixed the error reported here server-side. Clients will pick up the correct location after next synchronize. 

However, there was an additional error in MHAPI where the location for models were specified as [MH USER]/data/models rather than [MH USER]/models. This has been fixed in https://github.com/makehumancommunity/community-plugins-mhapi/commit/13ff0c86bb503dc04f0356d028383cba48f4cb55

So in order to get a working download for models, you'll also need to update MHAPI.
